docs/using.rst

```diff
@@ -87,6 +87,22 @@ Registering a New Version of a Schema Under the Subject "Kafka-value"
       {"id":1}
 
 
+---------------------------------------------------------------------
+Registering an Existing Schema to a New Subject Name
```

Comment:
Consider using simple present tense: "Register an existing schema...".

Reply:
@JimGalasyn : I generally agree with you but the entire rest of the page uses "-ing" so I was making it consistent with that.  And I didn't want to change them all because of existing linkages.  WDYT?

Reply:
Not a big deal, better to stay consistent in the page.